src/java/winstone/accesslog/SimpleAccessLogger.java

```diff
@@ -86,6 +87,17 @@ public void log(Request request, Response response) {
         synchronized (DF) {
             date = DF.format(new Date());
         }
+
+        // mimic https://github.com/eclipse/jetty.project/blob/jetty-9.4.0.v20161208/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractNCSARequestLog.java#L130
+        Authentication authentication = request.getAuthentication();
+        String remoteUser;
+        if (authentication instanceof Authentication.User) {
+            Authentication.User user = (Authentication.User) authentication;
+            remoteUser = user.getUserIdentity().getUserPrincipal().getName();
+        } else {
+            remoteUser = null;
```

Comment:
NIT: I'm wondering if logging then "NO USER" or something wouldn't be better. Maybe not and null is clearer in that regard.